### DISCUSSION TITLE
Open LocalHost Terminal Session

### DISCUSSION THREAD

**Discussion by iAGorynT** [isontheline/pro.webssh.net#1209] [General]
Hello.  Wondered if WebSSH can open a LocalHost Terminal connection to the computer I'm running it on, just like the Terminal App on a Mac that opens a local session showing a command prompt when started?  Haven't been able to run WebSSH without SSH, SFTP, or Telnet.

Any insights would be appreciated.

**Comment by isontheline** [comment]
Hello @iAGorynT 👋

Thanks a lot for your feedback!

Unfortunately WebSSH can't deal with local shells due to sandboxing but you can create a SSH connection to your own localhost if you want to. Just enable the SSH service on your Mac.

**Reply by iAGorynT** [reply]
Hello [isontheline](https://github.com/isontheline),

Thanks for your response.  I'll setup a LocalHost connection and all will be good!  

Really enjoying WebSSH and glad I upgraded to Pro.  Thank you for all your work, I really appreciate it.

Regards.

**Reply by isontheline** [reply]
Thank you so much @iAGorynT for your support about WebSSH 🙏
